resources/home/dnanexus/generate_fusion_workbook.py

```
@@ -0,0 +1,81 @@
+import dxpy
+
+from typing import List
+
+
+pip.main(["install", "--no-index", "--no-deps", *glob("packages/*")])
+
+import pandas as pd
+import openpyxl 
+
+from utils.utils import write_df_to_sheet
+from utils.fastqc import parse_fastqc
```

Ruff (0.8.2)

`utils.fastqc.parse_fastqc` imported but unused

Remove unused import: `utils.fastqc.parse_fastqc`

(F401)